src/libraries/System.Text.Json/tests/System.Text.Json.Tests/Serialization/EnumConverterTests.cs

```diff
@@ -135,17 +135,15 @@ public class Week
             [JsonConverter(typeof(JsonStringEnumConverter))]
             public DayOfWeek WorkStart { get; set; }
             public DayOfWeek WorkEnd { get; set; }
-            [LowerCaseEnum]
+            [JsonConverter(typeof(LowerCaseEnumConverter))]
             public DayOfWeek WeekEnd { get; set; }
         }
 
-        [AttributeUsage(AttributeTargets.Property | AttributeTargets.Class, AllowMultiple = false)]
-        private class LowerCaseEnumAttribute : JsonConverterAttribute
+        private class LowerCaseEnumConverter : JsonStringEnumConverter
```

Comment:
Can this class be `sealed` then?

Reply:
@EgorBo, you're asking about the test class?

Reply:
oops 😄